mass_mailing_newsletter_welcome_mail/i18n/mass_mailing_welcome_mail.pot

```diff
@@ -0,0 +1,35 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* mass_mailing_welcome_mail
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: mass_mailing_welcome_mail
+#: model:mail.template,body_html:mass_mailing_welcome_mail.email_template
+msgid "<div>Dear <b>${object.name}</b>,
+                    <br/>
+                    <br/>
+                    <h3>Thanks for signing up four our newsletter!</h3>
+                    <p>
+                        Enjoy a discount code for 20% off your first order
```

Comment:
You have to update the translations.

Reply:
Yes, with last change of this template doesn't update the .pot. It's already changed